extension/content.js

```diff
@@ -128,7 +128,7 @@ function addDeleteForkLink() {
 
 	if ($postMergeContainer.length > 0) {
 		const $postMergeDescription = $postMergeContainer.find('.merge-branch-description');
-		const forkPath = $postMergeContainer.attr('data-channel').split(':')[0];
+		const forkPath = `${getUsername()}/${repoName}`;
```

Comment:
Another completely different case here is renamed forks. You can fork a repo, rename the fork and it would still be a fork of the original. Example: https://github.com/hkdobrev/hkdobrev.github.io

I doubt people who have renamed a fork would want to have a delete button as they probably renamed it in order to keep it. Although we shouldn't generate 404s if possible I guess.

Keep in mind this is a rare case and shouldn't be addressed IMHO unless easy enough.